pages/dashboard/index.tsx

```diff
@@ -0,0 +1,60 @@
+import type {NextComponentType, NextPage} from 'next'
+import {
+    Avatar,
+    Box,
+    Button,
+    Menu,
+    MenuItem,
+    Toolbar,
+} from "@mui/material";
+import React from "react";
+
+import {usePopupState, bindTrigger, bindMenu} from 'material-ui-popup-state/hooks'
+import DBDrawerLayout from "../../components/layouts/dashboard/dbDrawer/dbDrawerLayout/DBDrawerLayout";
+
+
+const DashboardHome: NextPage = () => {
+
+    return (
+        <>
+            <DashboardAppBar/>
+
+            <Box sx={{ marginTop: 8,
+                display: 'flex',
+                flexDirection: 'column',
+                alignItems: 'center'}}>
+
+            </Box>
+        </>
+    )}
+
+const DashboardAppBar: NextComponentType = () => {
+
+    const AccountMenuPopupState = usePopupState({ variant: 'popover', popupId: 'accountMenu' })
+
+    return (
+        <>
+            <DBDrawerLayout/>
+
```

Comment:
We should probably instead use next layout provider but I can fix this in a bit if you don't want to

Reply:
yah just checkouted the commit you should def move the app bar for the dashboard into a component and then into Dashboard layout component then use the NextPage.getlayout to provide it like this
 `Page.getLayout = function getLayout(page: ReactElement) {
  return (
    <Layout>
      <NestedLayout>{page}</NestedLayout>
    </Layout>
  )
}`